apps/ledger-live-mobile/src/components/BleDevicePairingFlow/BleDevicePairing.tsx

```diff
@@ -22,7 +22,17 @@ export type BleDevicePairingProps = {
   deviceToPair: Device;
 };
 
-export const BleDevicePairing = ({
+/**
+ * Runs a BLE pairing with the given device. Displays pairing, success or error steps.
+ *
+ * A closing cross is displayed to the user during the pairing, which either calls onPaired if
+ * the device is already paired, or onRetry otherwise.
+ *
+ * @param deviceToPair Device to pair
```

Comment:
are you sure?

Reply:
🙈 📝

Reply:
it does mean i have good variable names 👑